src/utilities/validateUserInput.js

```diff
@@ -8,7 +8,9 @@ import isIdValid from './isValidId';
 const trimValues = (objectWithValuesToTrim) => {
   const trimmedValues = objectWithValuesToTrim;
   Object.keys(trimmedValues).forEach((key) => {
-    trimmedValues[key] = trimmedValues[key].trim();
+    if (key !== 'image') {
```

Comment:
What about other types? Do we only have sting and images?

Reply:
The initial validation fails for any key that that has a value of null which can be accepted but not for the image key.
So this fix allows the image key to accept null as an update because the user can decide to remove the image at any time

Reply:
The model will reject it and throw an error that will be caught and send back to the user because the model accepts only accept null or a URL

Reply:
I still don't understand this implementation. I'd like to see a demo @nwashangai

Reply:
Ok when would you like to see the demo?